### DISCUSSION TITLE
O que é necessário para começar um job remoto no exterior?

### DISCUSSION THREAD

**Discussion by camillodev** [frontendbr/forum#1191] [Discussão]
Oi Galera, 

To com a intenção de sair do pais, ir para Australia. E durante algum tempo, ficarei apenas com visto de estudante que so me permite trabalhar meio periodo. Então não será simples me sustentar haha. 
Pensando nisso, to querendo colocar outro sonho na pratica, que é de trabalhar remoto. Acho que todo desenvolvedor já sonhou pelo menos um pouco com isso, certo?
Eu até consigo arrumar um job do Brasil, porém não vale a pena pelo dolar. 
Sei que aqui na comunidade tem gente que trabalha remoto para empresas estrangeiras. 

Algumas Perguntinhas que seriam uteis saber, e acho que ajudara muita gente também:

- Conte um pouco a sua experiência se trabalha remotamente no brasil ou fora
- Usam "cnpj" do pais da empresa, ou se emitem nota fiscal com o cnpj daqui. 
- Se usam alguma plataforma para conseguir job
- Requisitos mais pedidos para front end
- Fazem freelancer ou trabalham para alguma empresa de forma fixa? 
- Dificuldades do trabalho remoto
- Acham importante ter um portfolio com blog para mostrar o que sabe ou apenas seus projetos é o suficiente?

Agradeço muito a ajuda galerinha

**Comment by jair-anderson-souza** [comment]
Dá uma olhada nessa issue #56,  pode ser útil pra você

**Comment by luizfelipehh** [comment]
@camillodev Interessante sua Issue, penso em ir para Austrália também, e principalmente trabalhar na área de desenvolvimento. É bastante complicado estudar e trabalhar no BR e quando quiser fazer algo do tipo assim se submeter a trabalhar com "qualquer coisa".

**Comment by camillodev** [comment]
vlw @jass2125 ja vi essa issue e ela é mto rica. vlw msm

**Comment by camillodev** [comment]
@luizfelipehh então cara eu to super disposto a trabalhar em qualquer coisa, ou os famosos "sub empregos" como chamamos aqui. Porém como vou com meu marido também, os custos de vida basicos e estudo ficaram altos. A maioria trabalha de forma ilegal lá, fazendo mais horas que permitido para os estudantes, eu vou evitar isso ao maximo. 
E a nossa area nos possibilita trabalhar remotamente, então acredito que seria a melhor forma de pagar meus estudos e custos la. Você ta planejando ir mesmo ou ta apenas como sonho? podemos trocar umas ideias depois.

**Comment by luizfelipehh** [comment]
@camillodev Estou planejando em ir sim! Porém me falta experiência em desenvolvimento web, concluo minha pós em WebDev e Mobile em Maio.

Pode me passar seu contato por e-mail?

**Comment by camillodev** [comment]
@luizfelipehh te add no linkedin e te passo email por la.

**Comment by rafaelwinter** [comment]
@camillodev Eu trabalho remotamente há pouco mais de um ano. 

A empresa para a qual trabalho não tem escritório no Brasil, fizemos um contrato de trabalho de acordo com as regras trabalhistas da Califórnia. Eu ainda não abri minha empresa, mas seria o ideal para perder menos dinheiro com impostos. Os impostos no meu caso precisam ser pagos avulsamente com o Carnê do Leão.

Eu não sei muito bem quais são os nossos requisitos para frontend, eu trabalho com Android e inclusive faço entrevistas técnicas com candidatos. Se precisar me adiciona no Linked In, posso te indicar lá: https://www.linkedin.com/in/rafaelwinter/

A vida inteira eu trabalhei em escritórios, e no momento em que eu comecei a trabalhar no meu próprio Home office, precisei me adaptar e criar uma rotina que me faz ser produtivo. Quando estamos em casa acabamos tendo um monte de distrações que não teríamos no escritório. Além disso, é muito comum ir passando da medida. Tipo... De repente começa a dar uma fome e você percebe que são 9 da noite já. kkk

**Comment by camillodev** [comment]
@rafaelwinter Brigadão viu, vou te adicionar sim no Linkedin. 
Já ouvi muito sobre a rotina e dificuldade de se adaptar ao novo estilo de trabalho que é home office, e ando lendo cada vez mais. Ando me adaptando a isso para futuramente poder trabalhar dessa forma. Quais praticas você aplicou no seu home office? 
A empresa em si, utiliza qual metodo de gestao para esse controle de produtividade?
Quais foram suas dificuldades? Senão se importar fala um pouco do seu processo.

**Comment by rafaelwinter** [comment]
@camillodev Eu acho que é fundamental ter um espaço separado, no meu caso uso um cômodo do apartamento só pra ser o escritório. É importante ter um ambiente separado do resto da casa, sem distrações, organizado para trabalhar e onde você se sinta bem. Porque isso te permite criar na cabeça uma rotina para ir trabalhar.

A empresa tem um sistema simples, nós registramos as horas trabalhadas em um sistema de controle de atividades. E com as nossas equipes, acertamos o que será entregue em cada sprint, nossos gerentes sabem como é a produtividade das pessoas, acompanham os problemas do projeto e vão sempre te acompanhar mais de perto se você tiver algum problema e sua produtividade cair.

**Comment by camillodev** [comment]
Bem interessante vou dar uma olhada na empresa. Brigadão pela ajuda

**Comment by felipefialho** [comment]
Linkando #56

**Comment by ghost** [comment]
Existe esse repositório com  uma lista de empresas de trabalho remoto. Referência: https://github.com/lerrua/remote-jobs-brazil

**Comment by kazzkiq** [comment]
@rafaelwinter Ia abrir uma Issue agora sobre esse assunto mas encontrei essa aqui e sua resposta muito me interessou por já ter trabalhado pra fora! 😄 

Surgiram umas dúvidas sobre a parte burocrática de receber valor do exterior trabalhando do Brasil. Consegue compartilhar com a gente o seu conhecimento nessa área? Vou separar em tópicos pra ficar mais organizado, tá?

- Como você fazia pra empresa te pagar? Passava só os dados da conta normal mesmo?
- Quanto de imposto foi cobrado? Tem alguma taxa específica pelo dinheiro vir do exterior?
- A conversão de moeda estrangeira pra BRL quem fazia era o próprio banco?
- Hoje, analisando com mais calma esse tempo trabalhando pra fora, acha que vale a pena?

Ficaria eternamente grato se pudesse ajudar com essas dúvidas, valeu!

**Comment by rafaelwinter** [comment]
Oi @kazzkiq, fico feliz em poder ajudar. Acho que o trabalho remoto veio para ficar, fazem três anos que trabalho exclusivamente assim, por isso acho muito importante informar ao máximo de pessoas sobre como funciona. Principalmente para quem trabalha para o exterior.

Sobre as dúvidas, vou tentar explicar ponto a ponto, de maneira detalhada.

**Recebendo pagamentos do exterior**
Existem algumas formas de receber seu pagamento feito do exterior. Vou falar das que considero as principais: receber diretamente pelo banco ou receber via serviços de pagamento. 

Para receber diretamente, seu banco precisa fornecer alguns dados como o código SWIFT da sua conta. Caso seu banco possa receber remessas internacionais, você informa o código SWIFT para a empresa pagadora e eles fazem a transferência (Wire transfer). Quando a transferência for recebida pelo seu banco, eles vão fazer uma operação de câmbio, para converter a moeda estrangeira em Reais.

Existem alguns inconvenientes em se receber diretamente no banco. A maioria dos bancos está (ou estava) pedindo o recibo (Invoice) referente ao pagamento. Criar o invoice é bem simples e tem um monte de modelos e sites que fazem isso. Mas todo mês tem que mandar o invoice, e em alguns casos entregar presencialmente na agência. Isso acaba gerando um atraso entre a data em que você recebe a transferência até o dia em que de fato o valor cai na sua conta.

A outra opção é receber através de uma empresa que processa pagamentos internacionais. 
Tem várias opções, como Payoneer, TransferWise, Husky, Remessa Online. Esses serviços vão pedir que você informe a eles sua conta nacional, e vão te fornecer um número de conta internacional. Quando a empresa pagadora deposita na sua conta internacional, você recebe um e-mail e pode converter esse valor para Reais e depositar na sua conta nacional.

Existem outras maneiras, criativas, de se receber pagamentos do exterior. Por exemplo, fora do sistema bancário, através de transações em criptomoedas. Mas isso fica numa zona cinzenta, não é legal nem ilegal, e pode causar problemas ao converter da criptomoeda para Reais por causa da volatilidade dos mercados. Isso eu não tentei, mas é possível, caso a empresa pagadora tope.

**Impostos**
A cobrança de impostos depende de como você fez seu contrato com a empresa pagadora. Se você fez por pessoa física, precisa pagar mensalmente seu imposto avulso utilizando o carnê do leão. Aqui é possível simular o valor dos impostos para PF: [Simulador do Carnê do Leão](http://www.receita.fazenda.gov.br/aplicacoes/atrjo/simulador/simulador.asp?tipoSimulador=M_Prox)

As regras mudam para pessoa jurídica, se você abrir uma empresa no simples nacional pode emitir notas com 6% de imposto, dependendo da atividade. Aí quando você for fazer retiradas de prolabore talvez pague impostos de PF nelas. Mas impostos menores do que pagaria se recebesse diretamente como pessoa física. Eu recomendo conversar com um contador para entender melhor os possíveis cenários para tentar pagar o mínimo de impostos dentro da lei. Existem várias coisas a se levar em conta, como o cálculo do Fator R, benefícios para exportadores de serviço e etc.

**Operação de câmbio**
Para converter um pagamento em moeda estrangeira, você faz uma operação de câmbio. Isso pode ser feito diretamente no banco, ou através de uma empresa que processe pagamentos, conforme eu expliquei no tópico sobre o recebimento. 

**A pergunta de um milhão de Reais (ou de Dólares): Vale a pena?**
TL;DR: vale sim.

Tem que se colocar tudo na balança. Receber seu salário em moeda estrangeira faz com que você não saiba quantos reais vão cair na sua conta mês que vem, por causa da volatilidade. Pode ser mais do que o normal, ou menos. Então é necessário ter disciplina e se acostumar a _sempre_ guardar uma parte do salário para evitar os riscos da variação cambial.

Você vai precisar lidar com pessoas que tem culturas, idiomas e maneiras de se expressar diferentes da nossa. O brasileiro é muito sensível a críticas e acaba sempre levando tudo para o lado pessoal, tipo "Ain o John falou na daily que meu último PR quebrou a compilação no CI e diminuiu a cobertura de testes. Não sei porque ele me odeia." É preciso vencer esse tipo de mentalidade para se trabalhar com estrangeiros.

A questão da disciplina é importante também. Para trabalhar em casa é ideal ter um cômodo separado, por causa das distrações. Esse cômodo é seu ambiente de trabalho, se acostume com horários para começar e principalmente para terminar de trabalhar. Nesse cômodo tenha uma mesa e uma cadeira que prestem, para que você tenha uma boa ergonomia para trabalhar sem se machucar.

Levando isso tudo em conta, pra mim está valendo a pena. Eu consigo almoçar com minha família, participo mais das coisas da casa, não perco tempo em transporte e tenho essa vivência de trabalhar com empresas de fora. Claro que a jornada muda de acordo com quem está caminhando, talvez tenham pessoas que não consigam se adaptar a esse modelo e aos riscos que ele traz.

**Comment by rafaelwinter** [comment]
@fabianoschaefer Olha, eu não entendo de offshores e não sei se, para isso que eu faço, é uma alternativa viável. Pelo que consta no artigo que você citou, no pior caso a gente pagaria $8000 de abertura + manutenção no primeiro ano. É bastante dinheiro, eu não sei se a economia em impostos chegaria a esse valor.

Além disso, eu não sei como se traz dinheiro de fora de outra maneira que não seja as que eu comentei. Não sei se, mesmo tendo uma offshore, você não precisaria ter uma PJ no Brasil para receber esse dinheiro.

Caso alguém aí tenha experiência com offshores, comente conosco!
